code/__HELPERS/global_lists.dm

```diff
@@ -12,11 +12,11 @@ var/global/list/active_diseases = list()
 var/global/list/events = list()
 
 //tips
-var/global/list/metatips = file2list('config/tips/metatips.txt')
-var/global/list/marinetips = file2list('config/tips/marinetips.txt')
-var/global/list/xenotips = file2list('config/tips/xenotips.txt')
-var/global/list/joketips = file2list('config/tips/joketips.txt')
-var/global/list/alltips = marinetips + xenotips + joketips + metatips
+GLOBAL_LIST_INIT(metatips, world.file2list("config/tips/metatips.txt"))
+GLOBAL_LIST_INIT(marinetips, world.file2list("config/tips/marinetips.txt"))
+GLOBAL_LIST_INIT(xenotips, world.file2list("config/tips/xenotips.txt"))
+GLOBAL_LIST_INIT(joketips, world.file2list("config/tips/joketips.txt"))
+GLOBAL_LIST_INIT(alltips, GLOB.marinetips + GLOB.xenotips + GLOB.joketips + GLOB.metatips)
```

Comment:
why?

`#define ALLTIPS GLOB.marinetips + GLOB.xenotips + GLOB.joketips + GLOB.metatips`

Reply:
I presume you want that line to be that.